Failing test: X-Pack Observability API Integration Tests.x-pack/solutions/observability/test/api_integration/apis/synthetics/sync_global_params·ts - Synthetics API Tests SyncGlobalParams parsed params for previously added http monitors

A test failed on a tracked branch

```
JestAssertionError: expect(received).toEqual(expected) // deep equality

- Expected  - 2
+ Received  + 2

  Object {
    "enabled": true,
    "policy_template": "synthetics",
    "streams": Array [
      Object {
        "compiled_stream": Object {
          "__ui": Object {
            "is_tls_enabled": false,
          },
          "check.request.body": "testValue",
          "check.request.headers": Object {
            "sampleHeader": "sampleHeaderValue",
          },
          "check.request.method": null,
          "check.response.status": Array [
            "200",
            "201",
          ],
          "enabled": true,
          "id": "5e479866-ba6d-403a-a1fd-b664335c0ac6",
          "ipv4": true,
          "ipv6": true,
          "max_attempts": 2,
          "max_redirects": 3,
          "mode": "any",
          "name": "test-monitor-name",
          "origin": "ui",
          "password": "test",
          "processors": Array [
            Object {
              "add_fields": Object {
                "fields": Object {
                  "config_id": "5e479866-ba6d-403a-a1fd-b664335c0ac6",
                  "meta": Object {
                    "space_id": "default",
                  },
                  "monitor.fleet_managed": true,
                },
                "target": "",
              },
            },
          ],
-         "proxy_url": "http://proxy.com",
+         "proxy_url": "${test}",
          "response.include_body": "never",
          "response.include_body_max_bytes": 1024,
          "response.include_headers": true,
          "run_from.geo.name": "Test private location 0",
          "run_from.id": "d3c3fc56-55e0-48a9-b9bc-cd9c94afc7b0",
          "schedule": "@every 5m",
          "tags": Array [
            "tag1",
            "tag2",
          ],
          "timeout": "180s",
          "type": "http",
          "urls": "https://nextjs-test-synthetics.vercel.app/api/users",
          "username": "test-username",
        },
        "data_stream": Object {
          "dataset": "http",
          "elasticsearch": Object {
            "privileges": Object {
              "indices": Array [
                "auto_configure",
                "create_doc",
                "read",
              ],
            },
          },
          "type": "synthetics",
        },
        "enabled": true,
        "id": "",
        "vars": Object {
          "__ui": Object {
            "type": "yaml",
            "value": "{\"is_tls_enabled\":false}",
          },
          "check.request.body": Object {
            "type": "yaml",
            "value": "\"testValue\"",
          },
          "check.request.headers": Object {
            "type": "yaml",
            "value": "{\"sampleHeader\":\"sampleHeaderValue\"}",
          },
          "check.request.method": Object {
            "type": "text",
            "value": "",
          },
          "check.response.body.negative": Object {
            "type": "yaml",
            "value": null,
          },
          "check.response.body.positive": Object {
            "type": "yaml",
            "value": null,
          },
          "check.response.headers": Object {
            "type": "yaml",
            "value": null,
          },
          "check.response.json": Object {
            "type": "yaml",
            "value": null,
          },
          "check.response.status": Object {
            "type": "yaml",
            "value": "[\"200\",\"201\"]",
          },
          "enabled": Object {
            "type": "bool",
            "value": true,
          },
          "id": Object {
            "type": "text",
            "value": "\"5e479866-ba6d-403a-a1fd-b664335c0ac6\"",
          },
          "ipv4": Object {
            "type": "bool",
            "value": true,
          },
          "ipv6": Object {
            "type": "bool",
            "value": true,
          },
          "location_id": Object {
            "type": "text",
            "value": "d3c3fc56-55e0-48a9-b9bc-cd9c94afc7b0",
          },
          "location_name": Object {
            "type": "text",
            "value": "\"Test private location 0\"",
          },
          "max_attempts": Object {
            "type": "integer",
            "value": 2,
          },
          "max_redirects": Object {
            "type": "integer",
            "value": "3",
          },
          "mode": Object {
            "type": "text",
            "value": "any",
          },
          "name": Object {
            "type": "text",
            "value": "\"test-monitor-name\"",
          },
          "origin": Object {
            "type": "text",
            "value": "ui",
          },
          "password": Object {
            "type": "password",
            "value": "\"test\"",
          },
          "processors": Object {
            "type": "yaml",
            "value": "[{\"add_fields\":{\"fields\":{\"monitor.fleet_managed\":true,\"config_id\":\"5e479866-ba6d-403a-a1fd-b664335c0ac6\",\"meta\":{\"space_id\":\"default\"}},\"target\":\"\"}}]",
          },
          "proxy_headers": Object {
            "type": "yaml",
            "value": null,
          },
          "proxy_url": Object {
            "type": "text",
-           "value": "\"http://proxy.com\"",
+           "value": "\"${test}\"",
          },
          "response.include_body": Object {
            "type": "text",
            "value": "never",
          },
          "response.include_body_max_bytes": Object {
            "type": "text",
            "value": "1024",
          },
          "response.include_headers": Object {
            "type": "bool",
            "value": true,
          },
          "schedule": Object {
            "type": "text",
            "value": "\"@every 5m\"",
          },
          "service.name": Object {
            "type": "text",
            "value": null,
          },
          "ssl.certificate": Object {
            "type": "yaml",
            "value": null,
          },
          "ssl.certificate_authorities": Object {
            "type": "yaml",
            "value": null,
          },
          "ssl.key": Object {
            "type": "yaml",
            "value": null,
          },
          "ssl.key_passphrase": Object {
            "type": "text",
            "value": null,
          },
          "ssl.supported_protocols": Object {
            "type": "yaml",
            "value": null,
          },
          "ssl.verification_mode": Object {
            "type": "text",
            "value": null,
          },
          "tags": Object {
            "type": "yaml",
            "value": "[\"tag1\",\"tag2\"]",
          },
          "timeout": Object {
            "type": "text",
            "value": "180s",
          },
          "type": Object {
            "type": "text",
            "value": "http",
          },
          "urls": Object {
            "type": "text",
            "value": "\"https://nextjs-test-synthetics.vercel.app/api/users\"",
          },
          "username": Object {
            "type": "text",
            "value": "\"test-username\"",
          },
        },
      },
    ],
    "type": "synthetics/http",
  }
    at comparePolicies (test_policy.ts:548:22)
    at Context.<anonymous> (sync_global_params.ts:262:22)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.apply (wrap_function.js:74:16) {
  matcherResult: {
    actual: {
      type: 'synthetics/http',
      policy_template: 'synthetics',
      enabled: true,
      streams: [Array]
    },
    expected: {
      type: 'synthetics/http',
      policy_template: 'synthetics',
      enabled: true,
      streams: [Array]
    },
    message: '\x1B[2mexpect(\x1B[22m\x1B[31mreceived\x1B[39m\x1B[2m).\x1B[22mtoEqual\x1B[2m(\x1B[22m\x1B[32mexpected\x1B[39m\x1B[2m) // deep equality\x1B[22m\n' +
      '\n' +
      '\x1B[32m- Expected  - 2\x1B[39m\n' +
      '\x1B[31m+ Received  + 2\x1B[39m\n' +
      '\n' +
      '\x1B[2m  Object {\x1B[22m\n' +
      '\x1B[2m    "enabled": true,\x1B[22m\n' +
      '\x1B[2m    "policy_template": "synthetics",\x1B[22m\n' +
      '\x1B[2m    "streams": Array [\x1B[22m\n' +
      '\x1B[2m      Obj
[report_failure] output truncated to 8192 characters
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/68551#0196f8b4-21b2-4186-8e88-74437dd562ac)

<!-- kibanaCiData = {"failed-test":{"test.class":"X-Pack Observability API Integration Tests.x-pack/solutions/observability/test/api_integration/apis/synthetics/sync_global_params·ts","test.name":"Synthetics API Tests SyncGlobalParams parsed params for previously added http monitors","test.failCount":1}} -->

--- a/x-pack/solutions/observability/test/api_integration/apis/synthetics/sync_global_params.ts
+++ b/x-pack/solutions/observability/test/api_integration/apis/synthetics/sync_global_params.ts
@@ -33,9 +33,7 @@ export const LOCAL_LOCATION = {
 };
 
 export default function ({ getService }: FtrProviderContext) {
-  // FLAKY: https://github.com/elastic/kibana/issues/221290
-  // Failing: See https://github.com/elastic/kibana/issues/221290
-  describe.skip('SyncGlobalParams', function () {
+  describe('SyncGlobalParams', function () {
     this.tags('skipCloud');
     const supertestAPI = getService('supertest');
     const kServer = getService('kibanaServer');
@@ -138,6 +136,7 @@ export default function ({ getService }: FtrProviderContext) {
           [ConfigKey.MONITOR_QUERY_ID]: apiResponse.body.id,
           [ConfigKey.CONFIG_ID]: apiResponse.body.id,
           locations: [LOCAL_LOCATION, pvtLoc],
+          spaces: ['default'],
         })
       );
       newMonitorId = apiResponse.rawBody.id;
@@ -173,6 +172,13 @@ export default function ({ getService }: FtrProviderContext) {
         .post(SYNTHETICS_API_URLS.PARAMS)
         .set('kbn-xsrf', 'true')
         .send({ key: 'test', value: 'http://proxy.com' });
+
+      /*
+       * Creating a global parameter kicks off an asynchronous background task.
+       * We pause for 5 seconds to let that task finish before creating monitors that reference the param;
+       * if it’s still running when the monitor is added, Kibana would not schedule a second sync task.
+       */
+      await new Promise((resolve) => setTimeout(resolve, 5000));
 
       expect(apiResponse.status).eql(200);
     });
@@ -237,6 +243,7 @@ export default function ({ getService }: FtrProviderContext) {
           [ConfigKey.MONITOR_QUERY_ID]: apiResponse.body.id,
           [ConfigKey.CONFIG_ID]: apiResponse.body.id,
           locations: [LOCAL_LOCATION, pvtLoc],
+          spaces: ['default'],
         })
       );
       newHttpMonitorId = apiResponse.rawBody.id;

